Row.c

```diff
@@ -428,23 +428,23 @@ void Row_printNanoseconds(RichString* str, unsigned long long totalNanoseconds,
    }
 
    unsigned long long totalSeconds = totalMicroseconds / 1000000;
-   unsigned long microseconds = totalMicroseconds % 1000000;
+   uint32_t microseconds = totalMicroseconds % 1000000;
    if (totalSeconds < 60) {
       int width = 5;
-      unsigned long fraction = microseconds / 10;
+      uint32_t fraction = microseconds / 10;
       if (totalSeconds >= 10) {
          width--;
          fraction /= 10;
       }
-      len = xSnprintf(buffer, sizeof(buffer), "%u.%0*lus ", (unsigned int)totalSeconds, width, fraction);
+      len = xSnprintf(buffer, sizeof(buffer), "%u.%0*lus ", (unsigned int)totalSeconds, width, (unsigned long)fraction);
       RichString_appendnAscii(str, baseColor, buffer, len);
       return;
    }
 
    if (totalSeconds < 600) {
-      unsigned int minutes = totalSeconds / 60;
-      unsigned int seconds = totalSeconds % 60;
-      unsigned int milliseconds = microseconds / 1000;
+      unsigned int minutes = (unsigned int)totalSeconds / 60;
```

Comment:
Shouldn't the cast here happen after the division?

Reply:
Not necessary because of the `(totalSeconds < 600)` conditional. Although I argue this is more of a style issue...
(Compiler should catch that `(unsigned int)(totalSeconds / 60)` can be optimized to `((unsigned int)totalSeconds) / 60`, no?)

Reply:
Not necessarily, as `(unsigned int)((1UL<<32) / 60)` != `(unsigned int)(1UL<<32) / 60`.

Won't happen in this case based on the previous check, but the correctness should take priority here.